APMrover2/Log.cpp

```diff
@@ -25,7 +25,8 @@ void Rover::Log_Write_Arm_Disarm()
 // Write an attitude packet
 void Rover::Log_Write_Attitude()
 {
-    const Vector3f targets(0.0f, 0.0f, 0.0f);  // Rover does not have attitude targets, use place-holder for commonality with Dataflash Log_Write_Attitude message
+    float desired_pitch_cd = degrees(g2.attitude_control.get_desired_pitch()) * 100.0f;
```

Comment:
Any real reason not to call the attitude controller for roll here as well?  It's probably just another way to get 0, but it seems neater.

Reply:
Txs for the review.  The attitude controller doesn't know about roll so it would be dead code so I'll leave this as-is.  We should possibly ask the attitude controller for the desired yaw and I thought about this but it's out-of-scope for this change I think.